src/java.base/unix/classes/sun/net/www/content-types.properties

```diff
@@ -395,3 +395,7 @@ application/bz2: \
 application/java-archive: \
 	description=JAR File;\
 	file_extensions=.jar;
+
+application/wasm: \
+	description=WASM File;\
```

Comment:
Perhaps the description could be `WebAssembly File;`? Same in the Windows' file.